src/datasets/inspect.py

```diff
@@ -303,6 +303,8 @@ def get_dataset_config_names(
      'rte',
      'wnli',
      'ax']
+
+    Note that you can fetch the list of configs for a dataset on the Hugging Face Hub via an HTTP request with the [Datasets REST API endpoint /splits](https://huggingface.co/docs/datasets-server/splits).
```

Comment:
I don't think adding this here would bring a lot of users. Maybe prefer adding some links in the actual docs